src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,86 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_ITEMS_QUANTITY = 10;
+    private Pair elements;
+    private int size = 0;
+    private Pair[] storage;
+
+    public StorageImpl() {
+        storage = new Pair[MAX_ITEMS_QUANTITY];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_ITEMS_QUANTITY; i++) {
```

Comment:
Use one` if` inside the loop
```suggestion
        for (int i = 0; i < size; i++) {
```

Reply:
I used size in loop, but do not know how to use 1 if.
I have 2 cases: 
 1st -  I need to add a new element to storage and increase size;
 2nd - I need to replace value in a storage, if one of storage keys equals to pair.key and do not to increase size. 
To do that i used else if in a loop.